users/tests.py

```diff
@@ -0,0 +1,155 @@
+from django.test import TestCase
+from users.models import CustomUser
+from django.urls import reverse
+from rest_framework.test import APITestCase
+from rest_framework.test import APIClient
+from django.core.exceptions import ValidationError
+from django.core.validators import EmailValidator
+from django.core.validators import MaxLengthValidator
+from users.common.exceptions import CustomValidationError
+from users.common.strings import CustomValidationErrorText
+from users.common import constants
+
+
+class TestCustomUserLogin(TestCase):
+    def setUp(self):
+        user = CustomUser.objects._create_user(
```

Comment:
This variable is not used anywhere.

Reply:
Done

Reply:
For testing data you can create `.yaml` fixtures.